jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>Something that I don't think is really mentioned or talked about much is cell-type specific transcriptional responses, and whether or not what scGen learns is a general perturbation response (aka the prediction is possible because it's the same across all cell types - which is still impressive) or it's adjusting for the cell type context (and therefore genes which are only upregulated in one cell type can be discovered). Unfortunately, I don't think there's much work done in quantifying exactly what is cell-type specificity... but anywho.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='104'/>

Reply:
<p>Fun question. I'll add some text for this.</p>